README.md

```diff
@@ -1 +1,42 @@
-# kubebuilder-plugin
\ No newline at end of file
+# Enable kubebuilder-plugin for operator-sdk
+
+
+- In order to use kubebuilder-plugin for java operators we need to clone the operator-sdk repo. Update the kubebuilder-plugin path in go-mod pointing to the local dir of your kube-builder repo. Example.
+
+```
+github.com/java-operator-sdk/kubebuilder-plugin => github.com/jmrodri/kubebuilder-plugin v0.0.0-20210317231109-2c27be7d4d7e
+```
```

Comment:
I want you to show them the exact steps because we don't want them pointing to my fork of the plugin. There are more steps to integrating the plugin with the SDK.  the replace line you put in is just a small piece. look at my java-plugin branch of the my operator-sdk fork. See the 2 commits and show them what they would have to do. You might have to try it yourself locally to see how it works if that helps.

https://github.com/jmrodri/operator-sdk/commits/java-plugin see the top 2 commits. There is a key piece they have to add to the `cli.go`.